test/functional/bundleremove/remove-client-certificate.bats

```diff
@@ -60,13 +60,6 @@ global_teardown() {
 		return
 	fi
 
-	destroy_web_server
```

Comment:
isn't destroying the web server still required though?

Reply:
Yes. I moved destroy_web_server into destroy_test_environment. This makes it easier to kill the web server with 'trap'.